sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/schema.yaml

```diff
@@ -164,6 +161,12 @@ fields:
 - name: website_channel_group
   type: STRING
   mode: NULLABLE
+- name: customer_start_date
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: end_date
+  type: TIMESTAMP
+  mode: NULLABLE
```

Comment:
This part isn't an actual change to the table schema, just updating the `schema.yaml` file to match the actual table schema as it hasn't been kept up to date.